instructions/kiali/kiali.md

```diff
@@ -21,7 +21,7 @@ Every pod deployed as part of the Service Mesh contains a proxy container respon
 Before we take a look at the Kiali console lets first generate some load on the Service Mesh.  From within a new terminal enter the following commands
 `export INGRESS_GATEWAY=$(oc get route -n istio-system istio-ingressgateway -o 'jsonpath={.spec.host}')`
 
-`while : do ; curl http://${INGRESS_GATEWAY}/ ; sleep 1 ; done`
+`while true; do curl http://${INGRESS_GATEWAY}; sleep 1; done`
 
 ### Opening the Kiali console
```

Comment:
The above is correct, 'while :' is the same as while true.

Reply:
The "while :" fails on both my rhel8 laptop and my mac with the same error on the "done".

Reply:
It's the ; not the 'while :' part, I've pushed up a separate PR with just this change since the java changes are to code we don't use.